### ISSUE TITLE
The same value predicted by trained neural network using Mxnet in R

### ISSUE THREAD

**Issue by Seager1989** [Seager1989/problem-notes#2]
Hello,
I am a beginner of deep learning and using Mxnet to train some model in R.

However, after I use the symbol method to create a neural network structure and trained it using `mx.model.FeedForward.create` , the model predicted the same value for different input values. Can someone help to offer some suggestions to fix this problem? This problem similar to another issue in [https://github.com/apache/incubator-mxnet/issues/5979](url)

I also attached the code below. Thanks

`library(mxnet)
library(readr)
DT <- read_csv("C:/Users/Administrator/Desktop/MXnet/3 ANN/Dataset for ANNOO.csv")

#####
ibrary(mxnet)
library(readr)
DT <- read_csv("C:/Users/Administrator/Desktop/MXnet/3 ANN/Dataset for ANNOO.csv")

######normalization
maxvalue<-apply(DT[,1:5],2,max)
minvalue<-apply(DT[,1:5],2,min)

DT[,1:5]<-as.data.frame(scale(DT[,1:5],center = minvalue,scale = maxvalue-minvalue))  #input normalization

ind<-sample.int(dim(DT),floor(0.8*(dim(DT)[1])))

DT<-as.data.frame(DT) 
mx.set.seed(0)

train.x=data.matrix(DT[ind,-6])  # Define the input data
train.y=DT[ind,6]        # Define the output data
test.x=data.matrix(DT[-ind,-6])
test.y=DT[-ind,6]
##define of ANN structures
data <- mx.symbol.Variable(name="data")
label = mx.symbol.Variable(name = 'label')
fc_layer = mx.symbol.FullyConnected(data = data, num.hidden = 8, name = 'fc_layer')
act_layer=mx.symbol.Activation(data = fc_layer, name = 'act_layer',act_type="tanh")
fc_layer1 = mx.symbol.FullyConnected(data = act_layer, num.hidden = 1, name = 'fc_layer1')
out_layer = mx.symbol.LinearRegressionOutput(data = fc_layer1, name = 'out_layer')

log <- mx.metric.logger$new()  # error vector

SEAmodel <- mx.model.FeedForward.create(out_layer, X=train.x, y=train.y,eval.data=list(data=test.x,label=test.y),
                                     ctx=mx.cpu(),num.round=1000,optimizer="sgd",array.batch.size=100,
                                     learning.rate=2e-2, momentum=0.9,eval.metric=mx.metric.rmse,
                                     epoch.end.callback = mx.callback.log.train.metric(period=5, log))   
preds = predict(SEAmodel, test.x)
preds
test.y
graph.viz(SEAmodel$symbol,shape=5,direction="TD",type="graph")
`

**Comment by Seager1989** [comment]
I try to normalize the train.y also. The result seems good. This might due to the different scale of the data between the input and output.
